client/reducers/cart.js

```diff
@@ -1,8 +1,35 @@
-const cartReducer = function(state={}, action) {
```

Comment:
My mistake. Lets rename this to order reducers.

I am not sure why we need GET_LINEITEMS, since LINEITEM just comes with the order.

Reply:
Is it because we need the product name? If so, we should do a nested include when we retrieve order detail.

Reply:
So the design is there are 3 states.
users (include order)
orders
products

so to access the cart, it would just be users active order. 
In order table, there will be isActive = t/f. That will help us determine.

When I built the skeleton I named it cart, it should just be orders.

Reply:
no, i was referring to the get_lineitems